components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIProvider.java

```diff
@@ -20,6 +20,7 @@
 import org.wso2.carbon.apimgt.api.dto.UserApplicationAPIUsage;
 import org.wso2.carbon.apimgt.api.model.*;
 
+import java.lang.reflect.InvocationTargetException;
```

Comment:
Any reason for importing this class. I can't see any changes other than this import in this class.